setup.cfg

```diff
@@ -54,7 +54,7 @@ install_requires =
 
 
 [options.package_data]
-* = *.json, *.pyi, py.typed, LICENSE
+* = *.json, *.template, *.pyi, py.typed, LICENSE
```

Comment:
Depending on how the proposal for `setuptools-scm` goes we might not need this one.